Cubical/Functions/Embedding.agda

```diff
@@ -58,6 +63,22 @@ injEmbedding {f = f} iSA iSB inj w x
   retr : retract (cong f) inj
   retr p = iSA w x _ p
 
+-- If `f` is an embedding, we'd expect the fibers of `f` to be
+-- propositions, like an injective function.
+hasPropFibers : (A → B) → Type _
+hasPropFibers f = ∀ y → isProp (fiber f y)
+
+-- This can be relaxed to having all prop fibers over the image, see [hasPropFibersOfImage→isEmbedding]
+hasPropFibersOfImage : (A → B) → Type _
+hasPropFibersOfImage f = ∀ x → isProp (fiber f (f x))
+
+-- some notation
+_↪_ : Type ℓ₁ → Type ℓ₂ → Type (ℓ-max ℓ₁ ℓ₂)
+A ↪ B = Σ[ f ∈ (A → B) ] isEmbedding f
```

Comment:
`hasPropFibers` -> `isEmbedding`. Less conversions in the other functions this way and more in line with the rest of the file.